README.md

```diff
@@ -31,31 +31,21 @@ src="https://img.shields.io/github/downloads/giovannicoppola/alfred-almanac/tota
 - Python3 (howto install [here](https://www.freecodecamp.org/news/python-version-on-mac-update/))
 
 ### Setup
-  
+
 1. Download the most recent release of `alfred-almanac` from Github and double-click to install
-2. In Alfred, open the 'Prepare workflow configuration and variables' window in `alfred-almanac` preferences
-	<img src='images/alfred5_prefs.png' width="500">
-			
-	- in the 'User Configuration' tab, set the 'Default Value' for the `LOCATION` variable to indicate the default locations as comma-separated city/town names, ZIP codes etc. 
-	- _Optional:_ in the 'User Configuration' tab, edit the 'Default Value' for the `FORMATSTRING` variable to customize the oneliner from `wttr.in`. Options and instructions available at the `wttr.in` [github page](https://github.com/chubin/wttr.in#one-line-output). 
-	- _Optional:_ in the 'User Configuration' tab, set the 'Default Value' for the `SPECIAL_DAY` variable (in the mm-dd format) to count days from and toward a special date. Default: `03-14` 
-	
+2. _Optional:_ Click `Configure Workflow` in `alfred-almanac` preferences to change settings
 3. _Optional:_ Setup a hotkey to launch alfred-almanac
-4. _Optional:_ Change the keyword to launch alfred-almanac
-	- keyword currently set to `!w`
-
-
 
 <a name="usage"></a>
```

Comment:
Why these links? They don’t seem to do anything.

Reply:
they are used by the index at the top of the readme file

Reply:
It’s not the links doing that, but the headers. The links at the top with a `#` are linking to specific IDs on the page. Those IDs are auto-generated by GitHub. Hover over any header in a GitHub README and you’ll see a small link icon.

Reply:
gotcha! but don't I need them if I want the links to work outside of Github, as a standalone markdown file?

Reply:
Ah, correct, for other platforms you may need it. But does that method work? I thought it should only work with `id`, not `name`.

Also, because an empty link is bound to not be semantic HTML, I’d argue for another solution. Instead of:

```html
<a name="usage"></a>
# Basic Usage
```

Having:

```html
<h1 id="basic-usage">Basic Usage</h1>
```

Fewer lines, semantic, and should work anywhere (untested).